packages/next-swc/docs/index.md

```diff
@@ -0,0 +1,7 @@
+# Turbopack
+
+Welcome to an overview of the Turbopack documentation.
+
+Currently it's work in progress, the only package available is [Turbo tasks-fs](turbo_tasks_fs/index.html)
+
+[comment]: # 'Temporary index page document'
```

Comment:
?

Reply:
It's a markdown comment. Will go away when updating docs.

Reply:
Why add it though

Reply:
To explicitly state index is work in progress temporary, just to render content. Could be a visible to the rendered page, in any case would like to note for anyone see this.